frontend/src/reviews/ReviewCommentDisplay.tsx

```diff
@@ -49,16 +83,33 @@ export default function ReviewCommentDisplay({ response, onReplyButtonClick }: R
               </IconButton>
             </Stack>
           </Stack>
-          {response.comment && (
-            <div>
-              <Divider sx={{ mt: 1, mb: 2 }} />
+          <Divider sx={{ mt: 1, mb: 2 }} />
+          {response.comment && !isEditMode && (
+            <Stack spacing={2}>
               <MarkdownDisplay>{response.comment}</MarkdownDisplay>
-            </div>
+              {response.updatedAt !== response.createdAt && (
+                <Typography variant='caption' sx={{ fontStyle: 'italic' }}>
+                  Updated {formatDateTimeString(response.updatedAt)}
+                </Typography>
+              )}
+            </Stack>
+          )}
+          {isEditMode && (
+            <>
+              <RichTextEditor value={updatedComment} onChange={(input) => setUpdatedComment(input)} />
+              <Box sx={{ float: 'right' }}>
+                <Button onClick={handleEditOnSave}>Save</Button>
+              </Box>
+              <Typography variant='caption' color={theme.palette.error.main}>
+                {editCommentErrorMessage}
+              </Typography>
+            </>
           )}
         </Card>
       </Stack>
       <Menu anchorEl={anchorEl} open={open} onClose={() => setAnchorEl(null)}>
         <MenuItem onClick={() => handleReplyOnClick(response.comment)}>Reply</MenuItem>
+        {currentUser && currentUser.dn === username && <MenuItem onClick={handleEditOnClick}>Edit comment</MenuItem>}
```

Comment:
Minor but I don't think specifying what's being edited is needed here. Consider changing the text to simply "Edit", but we'd need an `aria-label="Edit comment"` if you were to do that.

Reply:
I think this is fine for comment responses, but I think it's misleading for review responses. If I approve a release and I see the response at the bottom, and then click the menu icon and see "Edit", I'll assume that means I can also edit my response. Thoughts on keeping it as "Edit comment" for reviews?

Reply:
Yep totally agree! I hadn't thought about that.